frontend/src/Home.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
ooc, is this the same tests that catherine wrote? (like a duplication of her file)

Reply:
I have deleted this file. It is from when I pulled from master (which I now know not to do)